src/d3a/setup/power_flow/baseline_peak_energy.py

```diff
@@ -40,10 +40,6 @@ def get_setup(config):
                     grid_fee_percentage=0, grid_fee_constant=0,
                     baseline_peak_energy_import_kWh=0.4
                 ),
-                Area(
-                    'House 1 2', [],
-                    import_capacity_kVA=2.0, export_capacity_kVA=2.0
-                ),
```

Comment:
Can you please explain why you deleted this one here? Thanks

Reply:
Re-added